### PR TITLE
Add versionadded tag set_relative_mode

### PR DESCRIPTION
**Pull request by MyreMylar**

### CODE REVIEW

File: docs/reST/ref/mouse.rst [hunk 1]
```diff
@@ -241,6 +241,8 @@ scroll, such as ``which`` (it will tell you what exact mouse device trigger the
    Calling :func:`pygame.mouse.set_visible` with argument
    ``True`` will exit relative mouse mode.
 
+   .. versionadded:: 2.4.0
```
Comment (ankith26) [inline docs/reST/ref/mouse.rst:244]: needed for above function `get_relative_mode` too

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for the PR :star:

**Review by Matiiss — APPROVED** [review]
LGTM 👍

**Review by yunline — APPROVED** [review]
LGTM👍

**Pull request merged**